analysis/paper/paper.Rmd

```diff
@@ -40,15 +40,15 @@ These are our notes for preparing our slides for our CAA2018 presentation in the
 
 ## Slide 1: Title slide    
 
-In this presentation we will talk about how archaeologists are changing their data analysis practices to make their research more reproducible and open. We will the rrtools package, which is our specific contribution to help with the challenge of making reproducible research in archaeology easier.
+In this presentation we will talk about how archaeologists are changing their data analysis practices to make their research more reproducible and open. We will introduce the rrtools package, which is our specific contribution to help with the challenge of making reproducible research in archaeology easier. "Our" meaning in this case a group of people whose names you can see here and / or who took part in thethe 2017 Summer School on Reproducible Research in Landscape Archaeology at the Freie Universität Berlin (17-21 July), funded and jointly organized by Excellence Cluster 264 Topoi, the Collaborative Research Center 1266, and ISAAKiel under the guidance of Ben Marwick. 
 
```

Comment:
It's not really typical to make a statement like this in an academic presentation. The members of the audience assume that 'we', 'ours' refers to the people named on the first slide. 

Perhaps what we can do as a compromise is to move the acknowledgement slide to be the second slide. Then for slide two you can say this about the Summer School and its sponsors:

"Our group is names you can see here and / or who took part in the 2017 Summer School on Reproducible Research in Landscape Archaeology at the Freie Universität Berlin (17-21 July), funded and jointly organized by Excellence Cluster 264 Topoi, the Collaborative Research Center 1266, and ISAAKiel under the guidance of Ben Marwick."

To give the acknowledgements early like that is typical, and would work well here. What do you think? Of course you can add more names to that acknowledgements slide, I just put the people who mad PRs to the rrtools github repo.

Reply:
I will wait to hear what you think about this before merging this PR. Let me know!

Reply:
This is a good idea, let's do it on the second slide.